src/script/run-singularity.bash

```diff
@@ -9,13 +9,12 @@ echo $args
 ls
 if [ "$args" == "" ]; then args="/bin/bash"; fi
 
-if [[ "$(hostname -s)" =~ ^g[r,v,a] ]]; then nv="--nv"; fi
```

Comment:
@ajn313, can you rename this file so that it does not overwrite the existing greene launcher? Or you can use system variables if you want to use the same script.

Reply:
Done! Specified that this script is for XSEDE